src/main/java/command/DeleteCommand.java

```diff
@@ -0,0 +1,21 @@
+package main.java.command;
+
+import main.java.Task;
+import main.java.tasklist.TaskList;
+import main.java.ui.Ui;
+
+public class DeleteCommand extends Command {
+    private int deleteTaskIndex;
```

Comment:
deleteTaskIndex sounds more like a method name. Maybe something like indexToDelete?

Reply:
yepp it does sound like a method name